### PR TITLE
Add contact section

### PR DESCRIPTION
**Pull request by devMunyi**
-Added contact section which has:-
a heading, a description and the form itself

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,5 +252,65 @@ <h2 class="connect-text">LET'S CONNECT</h2>
...
+          <textarea
+            class="input message-input"
+            name="user_message"
+            id="user_message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          >
+Write your message here
```
Comment (ichala) [inline index.html:306]: - [ ] the `Write your message here` should be a placeholder, kindly remove it from here and add it as a placeholder attribute for the `<textarea>`

File: index.html [hunk 2]
```diff
@@ -242,5 +252,65 @@ <h2 class="connect-text">LET'S CONNECT</h2>
...
+
+      <p class="contact-deccription">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form
+        class="contact-form"
+        action="https://formspree.io/html/"
```
Comment (ichala) [inline index.html:268]: - [ ] kindly create an account at [formspree](https://formspree.io/) to get an id for your submission link, currently `https://formspree.io/html/` the `html` is not an id generated by formspree, so your form will not post the data the id should look like `https://formspree.io/xbs64f` (please do not use it it's just an example and you should get your own id )

File: index.html [hunk 3]
```diff
@@ -242,5 +252,65 @@ <h2 class="connect-text">LET'S CONNECT</h2>
...
+            rows="10"
+            maxlength="500"
+            required
+          >
+Write your message here
+      </textarea
+          ><br />
+        </label>
+
+        <button type="submit" class="contact-btn">Get It Touch</button>
```
Comment (ichala) [inline index.html:311]: - [ ] kindly make sure your button matches the Figma design and add the necessary hover, click and hover animations

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @devMunyi , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design ✅ 
- Correct workflow ✅ 

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -242,5 +252,63 @@ <h2 class="connect-text">LET'S CONNECT</h2>
...
+            name="user_message"
+            id="user_message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required></textarea
+          ><br />
+        </label>
+
+        <button type="submit" class="contact-btn">Get In Touch</button>
```
Comment (vickymarz) [inline index.html:309]: - [ ] As the previous reviewer suggested, kindly implement the button interactions such as the pressed and hover effect

When I hover on your submit button, the background color should turn sky-blue while the text turns white just as it is in the Figma design. Kindly add it to make your webpage nicer 😄 

Your design 👇🏽 
ezgif com-gif-maker

Figma Design 👇🏽 

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @devMunyi , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -242,5 +252,63 @@ <h2 class="connect-text">LET'S CONNECT</h2>
...
+            id="user_message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required></textarea
+          ><br />
+        </label>
+
+        <button type="submit" class="contact-btn btn">Get In Touch</button>
+      </form>
```
Comment (BenMukebo) [inline index.html:310]: - [ ] Great job so far on your form 👏🏻 However there are one missing requirement, kindly make sure to address this:

- As requested by the previous reviewer [here](https://github.com/devMunyi/Portfolio/pull/5#discussion_r952993475). Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.
 - Please use this image below 👇🏻 to improve the background color of your  `Get In Touch` button 👍🏻 

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @devMunyi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] This is a group project. Please let's include all the authors in the README file. The Getting Started section and the License section of [this template](https://github.com/microverseinc/readme-template) are missing in your README file. Please let's make sure that all the main headers in the template are present in your README file. This makes the README file professional and descriptive. 


- [OPTIONAL]  Please let's make sure that contributions to a group project are distributed equally among the members (commit messages should be fairly equal).


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (devMunyi) [reply index.html:310]: @BenMukebo which color is this in CSS?

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @devMunyi               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (BenMukebo) [reply index.html:310]: Hello @devMunyi, Sorry for the late reply, I can see that you made the changes already. Congratulations 🎊

**Pull request merged**